### ISSUE TITLE
AccountStore is null when ApplicationAccountStoreMapping is an Organization

### ISSUE THREAD

**Issue by dogeared** [stormpath/stormpath-sdk-java#433]
Setup an `Application` with only `Organization`s as its `AccountStore`s.

In the code below, `accountStore` is `null`

```
ApplicationAccountStoreMappingList accountStoreMappings = application.getAccountStoreMappings();
for (ApplicationAccountStoreMapping mapping : accountStoreMappings) {
    AccountStore accountStore = mapping.getAccountStore();
    if (!(accountStore instanceof Organization)) { continue; }

    Organization organization = (Organization) accountStore;
    System.out.println(organization.getName());
}
```

**Comment by maranai** [comment]
Pull request ready for review: https://github.com/stormpath/stormpath-sdk-java/pull/441

**Comment by dogeared** [comment]
closes #441
